.pipelines/.templates/vars.yml

```
@@ -4,13 +4,18 @@ variables:
   # Credentials
   # This reference is to the Variable Group which needs
   # to be created which will contain the following values.
+  #
+  # Set ARM_ENVIRONMENT to the Azure Environment you wish to use.
+  # Valid values are: AzureCloud, AzureChinaCloud, AzureGermanCloud, AzureUSGovernment
```

Even though it is technically a valid value, let's avoid mentioning AzureGermany since it is more or less decommissioned :) https://docs.microsoft.com/en-us/azure/germany/germany-migration-main
```suggestion
  # Valid values are: AzureCloud, AzureChinaCloud, AzureUSGovernment
```